metadata/service/server/init.yml

```diff
@@ -4,8 +4,8 @@ classes:
 - service.nagios.support
 parameters:
   _param:
-    nagios_password: r00tme
     nagios_username: nagiosadmin
+    nagios_password: r00tme
```

Comment:
And more generally it is not a good practice to define parameters in *_param* into service metadata right? I mean parameters under *_param* should be defined only at the system level?